src/solvers/localFixpoint.ml

```diff
@@ -0,0 +1,24 @@
+(** Local fixpoint iteration solvers that don't use a constraint system. *)
```

Comment:
In what sense is this solver local? There is only one unknown?

Reply:
In the sense that it's fixpoint iteration local to a single transfer function evaluation.

This is terminology from Miné (https://hal.sorbonne-universite.fr/hal-01657536), which originates from https://link.springer.com/chapter/10.1007/3-540-56287-7_95.

Reply:
Ah, ok, I was unaware that there are competing definitions of local in this space, our definition for local solver is that it only computes values for those unknowns needed for a query. Maybe this makes that clear?

```suggestion
(** Fixpoint iteration solvers local to a single transfer function (don't use a constraint system). *)
```